create_infotexts.py

```diff
@@ -30,14 +30,15 @@ def load_places_mapping():
     places_df = places_df.set_index("Nyckelord")
     places_df.columns = ["freq", "commonscat", "wikidata"]
 
-    places_df.replace("-", np.nan)
+    places_df[places_df == "-"] = None
```

Comment:
OK. Me being confused about it being a panda object.

Then just fix the typo on L29 ("**is** a dataframe") and insert a blank line after L23